Documentation/Contributing/Roadmap.md

```diff
@@ -23,67 +26,106 @@ The Mixed Reality Toolkit will require Unity 2018.4.
 
 > When Unity releases an LTS (Long Term Support) product, the Mixed Reality Toolkit will update to the LTS release. MRTK will also support the latest non-beta (ex: 2019.1) tech branch version of Unity, at the time at which MRTK was released.
 
-### 2.3.0
+### 2.4.0
 
-The version 2.3.0 plan has been finalized. While some details may change as the iteration progresses, the following describes the overall plan for the next MRTK release. For the latest status of the release, please visit the [milestone page]( https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/10).
+For the latest status of the release, please visit the [milestone page]( https://github.com/microsoft/MixedRealityToolkit-Unity/milestone/11).
 
 Status: In development
 
-Timeline: January 2029
+Timeline: March 2020
 
 Themes:
 
 - Stability
 - Developer education
-- Unity 2019.3 XR SDK support
 - User Experience
-- Iterate on mobile AR support
+- Hand Extensibility
 
 **Stability**
 
 Quality and stability are the top priority for this and all Microsoft Mixed Reality Toolkit releases. We will continue to prioritize customer and partner issues that impact the stability of MRTK components.
 
-The MRTK build and deploy tools remain a continued area of stability and quality focus.
-
 **Developer education**
 
 [Developer documentation](https://microsoft.github.io/MixedRealityToolkit-Unity) and example scenes are, like stability, an ongoing priority for the MRTK team.
 
-**Unity 2019.3 XR SDK support**
+**User Experience**
 
-XR SDK is Unity's new mixed reality platform. The Microsoft Mixed Reality Toolkit is committed to supporting our customers on this new platform. All new platform support in MRTK is expected to be delivered via XR SDK.
+- Bug fixes
+- Hololens Shell parity
+- Graduating experimental features
+- Tests to ensure features do not regress
 
-**User Experience**
+**Hand extensibility**
```

Comment:
What's your bar for things that should be called out here, vs not called out here? It feels like there are things in the UX world (@julenka) that I feel like should be called out at a higher level before this would be there.

Reply:
I believe most of "hand extensibility" discussions with Dave and @keveleigh  center around adding support for oculus quest. In that case, if we already mention adding quest support, and in particular quest hand tracking support, I would vote to remove this line as it feels duplicate (we already need to make hands extensible if we are adding quest support for hands).

Regarding adding more specifics to UX features, could we please first do a planning meeting before adding any more detail? Ping @thalbern  as FYI so she is aware of these points we are adding to public docs.

Reply:
I generally add things that are a significant amount of work that we are committed to deliver.

@julenka, if you would like more detail under UX, please feel free to commit to this change or mention them here. whatever works best for you.

Reply:
will remove that line. agree it seems redundant

Reply:
have replaced it with "Platform Expansion: Hands" and discussion of Leap Motion and Oculus Quest